source/dubregistry/registry.d

```diff
@@ -248,6 +277,62 @@ class DubRegistry {
 		return ret;
 	}
 
+	PackageInfo[string] getPackageInfos(string[] packnames, PackageInfoFlags flags)
+	{
+		import dub.recipe.packagerecipe : getBasePackageName;
+
+		PackageInfo[string] infos;
+		void[0][string] visited;
```

Comment:
Walter wants to deprecate this. 

https://github.com/dlang/dmd/pull/7986

Why not `bool`?